src/RawRabbit.Enrichers.MessagePack/MessagePackPlugin.cs

```diff
@@ -0,0 +1,19 @@
+﻿using RawRabbit.Instantiation;
+using RawRabbit.Serialization;
+
+namespace RawRabbit.Enrichers.MessagePack
+{
+	public static class MessagePackPlugin
+	{
+		/// <summary>
+		/// Replaces the default serializer with MessagePack.
+		/// </summary>
+		public static IClientBuilder UseMessagePack(this IClientBuilder builder, bool useLz4 = false)
```

Comment:
Instead of a `bool` here, perhaps create an `Compression` enum that can be used for the compression type. That enum, un turn, could be used to register the serializer in the ioc container, making it possible to inject it in the `ISerializer` (see comment below)

Reply:
I have made a descriptive enum to replace the boolean. 

I'm unsure how the enum would fit in the IOC container.